api/openapi-spec/swagger.json

```diff
@@ -12543,6 +12543,10 @@
    "v1.ACPI": {
     "type": "object",
     "properties": {
+     "msdmNameRef": {
```

Comment:
Can we squash b7c109cad745e2bf266dc07e31d0c055fc761d7f ?

Reply:
Yes!, no problem

Reply:
/hold let me squash it before merge

Reply:
/unhold